rotkehlchen/tests/api/blockchain/test_evm.py

```diff
@@ -235,6 +241,23 @@ def test_add_multievm_accounts(rotkehlchen_api_server):
     )
     assert_proper_response(response)
 
+    # patch modify_blockchain_accounts to check that we handle correctly failures when
+    # adding new accounts
+    original_modify_blockchain_accounts = rotki.chains_aggregator.modify_blockchain_accounts
+
+    def new_modify_blockchain_accounts(blockchain, accounts, append_or_remove) -> Callable:  # pylint: disable=unused-argument
+        """Make fail the logic for adding new accounts if  failing_account is given as argument"""
```

Comment:
please notice the removal of the extra whitespace

```suggestion
        """Make the logic fail when adding new accounts if failing_account is given as argument"""
```